src/plugin.ts

```diff
@@ -40,6 +42,21 @@ export class PluginTypeEnvironment extends TypeEnvironment {
         return this.getType(this.checker.lookupTypeAt(n, this.sinapSourceFile));
     }
 
+    private getFunctionSignatures(name: string, node: ts.Node) {
+        const functionSymbol = this.checker.getSymbolsInScope(node, ts.SymbolFlags.Function)
+            .filter((a) => a.name === name)[0];
+        if (functionSymbol === undefined) {
+            throw new Error(`function "${name}" not found`);
```

Comment:
Throwing an error gives much better exception trackbacks in angular and anywhere except chrome (which handles strings magically, but no better that Error objects).